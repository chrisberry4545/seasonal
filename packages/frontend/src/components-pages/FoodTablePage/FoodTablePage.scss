@import '../../styles/settings';
@import '../../styles/tools';

  .c-food-table-page__main {
    width: 100%;
  }
