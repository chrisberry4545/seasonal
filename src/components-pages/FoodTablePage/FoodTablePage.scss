@import '../../styles/settings';
@import '../../styles/tools';

.c-food-table-page {
  margin-top: $default-top-margin;
  @include mq($from: tablet) {
    display: flex;
  }
}

  .c-food-table-page__main {
    width: 100%;
  }
