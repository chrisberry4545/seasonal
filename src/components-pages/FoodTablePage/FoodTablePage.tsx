import React from 'react';

import './FoodTablePage.scss';

import {
  BottomTabsConnecter,
  CurrentSeasonNameConnecter,
  FoodTableConnecter,
  HeaderConnecter,
  SeasonMenuConnecter
} from '../../components-main';

export const FoodTablePage = () => (
  <div className='c-food-table-page'>
    <HeaderConnecter />
    <SeasonMenuConnecter />
    <div className='c-food-table-page__main'>
      <CurrentSeasonNameConnecter />
      <FoodTableConnecter />
    </div>
    <BottomTabsConnecter />
  </div>
);
