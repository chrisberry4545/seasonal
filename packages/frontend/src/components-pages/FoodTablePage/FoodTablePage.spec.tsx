import React from 'react';
import { shallow, ShallowWrapper } from 'enzyme';
import { FoodTablePage } from '../FoodTablePage/FoodTablePage';

jest.mock('../../components-main', () => ({
  BottomTabsConnecter: () => 'BottomTabsConnecter',
  CurrentSeasonNameConnecter: () => 'CurrentSeasonNameConnecter',
  FoodTableConnecter: () => 'FoodTableConnecter',
  PageWithMenu: () => 'PageWithMenu',
  RecipeTableConnecter: () => 'RecipeTableConnecter'
}));

describe('<FoodTablePage />', () => {
  let wrapper: ShallowWrapper;

  beforeEach(() =>
    wrapper = shallow(
      <FoodTablePage />
    )
  );

  test('renders correctly', () => expect(wrapper).toMatchSnapshot());

});
