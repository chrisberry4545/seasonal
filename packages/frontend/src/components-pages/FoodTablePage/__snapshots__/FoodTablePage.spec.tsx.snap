// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FoodTablePage /> renders correctly 1`] = `
<PageWithMenu>
  <div
    className="c-food-table-page__main"
  >
    <CurrentSeasonNameConnecter />
    <FoodTableConnecter />
    <RecipeTableConnecter />
    <BottomTabsConnecter />
  </div>
</PageWithMenu>
`;
